app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ] From the project requirement, you should have `update_comments_counter` method and make it private.

def update_comments_counter
  post.increment!(:comments_counter)
end

- [ ] Also I recommend you use `Rails Active Callbacks`- `after_save` to call that update method every time a new comment is saved to database.

 class Comment < ApplicationRecord
   ` after_save : update_comments_counter`
....
    private
     def update_comments_counter
      ...
    end
end